node/resolvers/masterdata/index.ts

```diff
@@ -1,19 +1,51 @@
 import http from 'axios'
 import paths from '../paths'
-import {map, keys} from 'ramda'
+import {zipObj, mergeAll} from 'ramda'
 import { withAuthToken, headers } from '../headers'
 
+/**
+ * Map a document object to a list of {key: 'property', value: 'propertyValue'}.
+ */
+const mapKeyValues = (document) => Object.keys(document).map(key => ({
+  key,
+  value: document[key]
+}))
+
+/* 
+ * Convert a list of fields like [ {key: 'propertyName', value: 'String'}, ... ]
+ * to a JSON format. 
+ */
+const parseFieldsToJson = (fields) => mergeAll(
+  fields.map(field => zipObj([field.key], [field.value]))
+)
+
 export const queries = {
-  masterObjects: async (_, args, { vtex: ioContext, request: {headers: {cookie}}}) => {
+  documents: async (_, args, { vtex: ioContext, request: {headers: {cookie}}}) => {
```

Comment:
Don't you think that is time to create a `documentsResolver.ts` ?

Reply:
👍